### PR TITLE
Issues 5 Vlad Polivko

### PR DESCRIPTION
**Pull request by botonojke**
Add models UniqBoltTransactions, UniqUberTransactions 
First you need to do makemigrations

Added scripts for saving unique transactions to the database(Bolt, Uber)
How to run scripts:
For Bolt - docker exec -it universal_auto_web python manage.py runscript bolt_transactions
For Uber - docker exec -it universal_auto_web python manage.py runscript uber_transactions
Then you need to pass the name of the csv file for Bolt or Uber

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class BoltPaymentsOrder(models.Model):
     report_to = models.DateTimeField()
     report_file_name = models.CharField(max_length=255)
     driver_full_name = models.CharField(max_length=24)
-    mobile_number = models.CharField(max_length=13)
```
Comment (SergeyKutsko) [inline app/models.py:59]: shoul be 13

File: app/models.py [hunk 2]
```diff
@@ -150,5 +150,105 @@ def save_uber_report_to_db(file_name):
                                  ajustment_payment =row[20],
                                  cancel_payment = row[21])
             order.save()
-    
-  
+
+class UniqUberTransactions(models.Model):
```
Comment (SergeyKutsko) [inline app/models.py:154]: Rename to UberTransactions

File: app/models.py [hunk 3]
```diff
@@ -150,5 +150,105 @@ def save_uber_report_to_db(file_name):
...
+                                                   transfered_to_bank=row[17],
+                                                   peak_rate=row[18],
+                                                   tips=row[19],
+                                                   cancel_payment=row[20])
+                transaction.save()
+            except IntegrityError:
+                print(f"{row[0]} transaction is already in DB")
+
+
+class UniqBoltTransactions(models.Model):
```
Comment (SergeyKutsko) [inline app/models.py:210]: rename to BoltTransactions(

File: app/models.py [hunk 4]
```diff
@@ -150,5 +150,105 @@ def save_uber_report_to_db(file_name):
...
+    paymant_authorization = models.DecimalField(decimal_places=2, max_digits=10)
+    service_tax = models.DecimalField(decimal_places=2, max_digits=10)
+    cancel_payment = models.DecimalField(decimal_places=2, max_digits=10)
+    tips = models.DecimalField(decimal_places=2, max_digits=10)
+    order_status = models.CharField(max_length=50)
+    car = models.CharField(max_length=50)
+    license_plate = models.CharField(max_length=30)
+
+    class Meta:
+        unique_together = (('driver_name', 'trip_date', 'paymant_conformation', 'boarding'))
```
Comment (SergeyKutsko) [inline app/models.py:228]: Should be uniq by telephone as well. you have some bad eng word paymant_conformation

File: app/models.py [hunk 5]
```diff
@@ -150,5 +150,105 @@ def save_uber_report_to_db(file_name):
...
+    fare2 = models.DecimalField(decimal_places=2, max_digits=10)
+    service_tax = models.DecimalField(decimal_places=2, max_digits=10)
+    wait_time = models.DecimalField(decimal_places=2, max_digits=10)
+    transfered_to_bank = models.DecimalField(decimal_places=2, max_digits=10)
+    peak_rate = models.DecimalField(decimal_places=2, max_digits=10)
+    tips = models.DecimalField(decimal_places=2, max_digits=10)
+    cancel_payment = models.DecimalField(decimal_places=2, max_digits=10)
+
+
+def save_uber_transactions_to_db(file_name):
```
Comment (SergeyKutsko) [inline app/models.py:178]: transaction_uuid should be uniq constraint on DB level

**Comment by SergeyKutsko** [comment]
@botonojke add tests

File: app/models.py (hunk 5 above)
Comment (botonojke) [reply app/models.py:178]: transaction_uuid = models.UUIDField(unique=True) -  unique constraint at the DB level

**Comment by SergeyKutsko** [comment]
@botonojke please resolve conflicts

**Pull request closed without merging**